README.md

```diff
@@ -14,24 +14,24 @@ services:
 
 ## Environment Variables
 
-| Name                        | Description                                                                   | Type      |
-| --------------------------- | ----------------------------------------------------------------------------- | ---------
-| `MU_SPARQL_ENDPOINT`        | The endpoint that will receive SPARQL queries                                 | UrlString |
-| `MU_APPLICATION_GRAPH`      | Graph containing the cleanup jobs                                             | UrlString |
-| `CRON_PATTERN`              | Default cron pattern for cleanup jobs that do not define one                  | String    |
-| `PING_DB_INTERVAL`          | Interval (ms) to wait before pinging to confirm if the database is running    | Int       |
-| `SCHEDULE_ON_SERVICE_START` | Allows the cleanup jobs to be automatically scheduled when the service starts | Bool      |
+| Name                        | Description                                                                   | Type      | Default value                      |
+| --------------------------- | ----------------------------------------------------------------------------- | --------- | ---------------------------------- |
+| `MU_SPARQL_ENDPOINT`        | The endpoint that will receive SPARQL queries                                 | UrlString | 'http://database:8890/sparql'      |
+| `MU_APPLICATION_GRAPH`      | Graph containing the cleanup jobs                                             | UrlString | 'http://mu.semte.ch/graphs/public' |
+| `CRON_PATTERN`              | Default cron pattern for cleanup jobs that do not define one                  | String    | '0 20 1 * *'                       |
+| `PING_DB_INTERVAL`          | Interval (sec) to wait before pinging to confirm if the database is running    | Int       | 2                                  |
```

Comment:
The `PING_DB_INTERVAL` has an extra space in the `type` column, which makes it unaligned with the rest of the table:
```suggestion
| `PING_DB_INTERVAL`          | Interval (sec) to wait before pinging to confirm if the database is running   | Int       | 2                                  |
```